app/sockets/service.py

```diff
@@ -101,7 +101,33 @@ def create_conversation(conversation_id, creator, participants):
                 result.append(participant)
                 
         return result
+
+    @staticmethod
+    def remove_global_identifier(email):
```

Comment:
Maybe rename this to `remove_identifier` because this will remove both `global` and `conversation` from the mapping